native/cocos/bindings/manual/jsb_xmlhttprequest.cpp

```diff
@@ -576,6 +587,7 @@ se::Class *__jsb_XMLHttpRequest_class = nullptr; //NOLINT(readability-identifier
 static bool XMLHttpRequest_finalize(se::State &s) { //NOLINT(readability-identifier-naming, google-runtime-references)
     auto *request = static_cast<XMLHttpRequest *>(s.nativeThisObject());
     SE_LOGD("XMLHttpRequest_finalize, %p ... \n", request);
+    request->clearCallbacks();
```

Comment:
From v3.6 , we modify the behavior of the lifecycle of c++ object which is controlled by se::Object now rather than doing `release` or `delete` in finalizer.